docs/Contributing/architecture/software/README.md

```diff
@@ -0,0 +1,21 @@
+# Software Architecture
+
+This directory contains documentation about Fleet's Software architecture.
+
+## Overview
+
+Fleet's Software architecture is designed to manage software across the device fleet, including software inventory, vulnerability management, and software installation. This documentation provides insights into the design decisions, system components, and interactions specific to the Software functionality.
```

Comment:
```suggestion
Fleet's Software architecture is designed to manage software across the device fleet, including software inventory, vulnerability management, and software installation. This directory contains documentation about Fleet's Software architecture.
```

Going to read over the software articles more closely to ensure we're getting more concise docs. I assume someone else will do the same for orch/MDM; my comments there were from a quick skim rather than an in-depth read.

Reply:
Yep! I asked each EM to review their architecture directory.